ocaml/xapi/api_server.ml

```diff
@@ -122,7 +122,7 @@ let is_himn_req req =
   | None -> false
 
 (* This bit is called directly by the fake_rpc callback *)
-let callback1 is_json req fd call =
+let callback1 is_json is_jsonrpc req fd call =
```

Comment:
can we add a comment explaining why we need  both `is_json` and `is_jsonrpc`? on the surface it looks like accidental complexity

Reply:
It is accidental complexity :) But sure, I'll add a comment.